src/Data/Product/Properties.agda

```diff
@@ -53,6 +55,9 @@ module _ {B : A → Set b} where
 ,-injective : ∀ {a c : A} {b d : B} → (a , b) ≡ (c , d) → a ≡ c × b ≡ d
 ,-injective refl = refl , refl
 
+map-cong₂ : ∀ {f g : A → C} {h i : B → D} → f ≗ g → h ≗ i → map f h ≗ map g i
```

Comment:
Hmm I think this should be called just `map-cong`? And then the subscripts can be reserved for mapping over either the first or second projections?

Reply:
I think you might be right. Data.Sum.Properties uses that naming convention.